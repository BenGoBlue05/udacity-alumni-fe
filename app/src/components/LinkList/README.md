## LinkList Component
A component that ...

### Example

```js
<LinkList />
```

### Props

| Prop          | Type     | Default     | Possible Values
| ------------- | -------- | ----------- | ---------------------------------------------
| **myProp**    | String   |             | Any string value


### Other Information
