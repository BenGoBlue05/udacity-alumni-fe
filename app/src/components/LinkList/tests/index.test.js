import LinkList from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<LinkList />', () => {
  it('should render with default props', () => {
  	const links = ['react', 'graphql', 'ruby'];
  	const separator = ',';
    const wrapper = shallow(
      <LinkList links={links} separator={separator} />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
