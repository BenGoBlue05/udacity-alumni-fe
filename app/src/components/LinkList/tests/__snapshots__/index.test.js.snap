exports[`<LinkList /> should render with default props 1`] = `
<Box
  announce={false}
  direction="row"
  focusable={true}
  pad="none"
  responsive={false}
  tag="div"
  wrap={true}>
  <span>
    <Anchor
      href="/archive?tag=undefined"
      label="undefined"
      tag="a" />
    , 
  </span>
  <span>
    <Anchor
      href="/archive?tag=undefined"
      label="undefined"
      tag="a" />
    , 
  </span>
  <span>
    <Anchor
      href="/archive?tag=undefined"
      label="undefined"
      tag="a" />
  </span>
</Box>
`;
